index.html

```diff
@@ -9,691 +9,80 @@
+    <script class="remove">
+      var respecConfig = {
+          specStatus: "ED",
+	  shortName: "PNG",
+	  subtitle: "Information technology — Computer graphics and image processing — Portable Network Graphics (PNG): Functional specification. ISO/IEC 15948:2003 (E)",
+	  copyrightStart: "1996",
+	  previousPublishDate: "2003-11-10",
+	  previousMaturity: "REC",
+	  edDraftURI: "https://github.com/w3c/PNG-spec",
+	  extraCSS: ["http://dev.w3.org/2009/dap/ReSpec.js/css/respec.css"],
+	  editors: [
+	      { name: "Chris Blume", url: "https://www.programmax.net", w3cid: 127631 },
+	      { name: "Pierre-Anthony Lemieux", company: "Sandflow Consulting LLC", companyURL: "https://www.sandflow.com" },
+	      { name: "Chris Lilley", url: "https://svgees.us", company: "W3C", companyURL: "https://www.w3.org" },
+	      { name: "Leonard Rosenthol", company: "Adobe Inc.", companyURL: "https://www.adobe.com" },
+	      { name: "Chris Seeger", company: "Comcast Corporation", companyURL: "https://www.comcast.com" }
+	  ],
+	  formerEditors: [
+	      { name: "Thomas Boutell" },
+	      { name: "Adam M. Costello" },
+	      { name: "David Duce" },
+	      { name: "Tom Lane" },
+	      { name: "Glenn Randers-Pehrson" },
+	      { name: "Oxford Brookes University (Second Edition)" }
+	  ],
+	  authors: [
+	      { name: "Mark Adler", url: "http://www.alumni.caltech.edu/~madler/" },
+	      { name: "Thomas Boutell", url: "http://www.boutell.com/boutell/" },
+	      { name: "John Bowler" },
+	      { name: "Christian Brunschen", url: "http://www.df.lth.se/~cb/" },
+	      { name: "Adam M. Costello", url: "http://www.nicemice.net/amc/" },
+	      { name: "Lee Daniel Crocker", url: "http://www.piclab.com/" },
+	      { name: "Andreas Dilger", url: "http://www-mddsp.enel.ucalgary.ca/People/adilger/" },
+	      { name: "Oliver Fromme", url: "http://www.fromme.com/" },
+	      { name: "Jean-loup Gailly", url: "http://www.teaser.fr/~jlgailly/" },
+	      { name: "Chris Herborth" },
+	      { name: "Alex Jakulin" },
+	      { name: "Neal Kettler" },
+	      { name: "Tom Lane" },
+	      { name: "Alexander Lehmann" },
+	      { name: "Chris Lilley", url: "https://svgees.us", company: "W3C", companyURL: "https://www.w3.org" },
+	      { name: "Dave Martindale" },
+	      { name: "Owen Mortensen" },
+	      { name: "Keith S. Pickens" },
+	      { name: "Robert P. Poole", url: "http://www.users.qwest.net/~lionlad/" },
+	      { name: "Glenn Randers-Pehrson" },
+	      { name: "Greg Roelofs", url: "http://pobox.com/~newt/" },
+	      { name: "Willem van Schaik", url: "http://www.schaik.com/" },
+	      { name: "Guy Schalnat" },
+	      { name: "Paul Schmidt" },
+	      { name: "Michael Stokes" },
+	      { name: "Tim Wegner" },
+	      { name: "Jeremy Wohl" }
+	  ],
+	  wg: "Portable Network Graphics (PNG) Working Group",
+	  wgURI: "https://www.w3.org/groups/wg/png",
+	  wgPublicList: "public-png",
+	  wgPatentURI: "",
```

Comment:
Should be https://www.w3.org/groups/wg/png/ipr

Reply:
Fixed.